ADAL/src/request/ADWebAuthResponse.m

```diff
@@ -394,7 +394,7 @@ - (void)handleJSONError:(NSError*)jsonError
         }
         
         AD_LOG_ERROR(_request.correlationId, @"JSON deserialization error:");
-        AD_LOG_ERROR_PII(_request.correlationId, @"%@ - %@", jsonError.description, bodyStr);
+        AD_LOG_ERROR_PII(_request.correlationId, @"JSON deserialization error: %@ - %@", jsonError.description, bodyStr);
```

Comment:
can jsonError.description contain PII? if not, we should keep it in non-PII

Reply:
This is a part of comment above that line: 

// We're often seeing the JSON parser being asked to parse whole HTML pages.
// Logging out the whole thing is unhelpful as it contains no useful info.

I prefer to mark it as PII, since we can get any text there and it might contain PII. @oldalton @RPangrle @jasoncoolmax @unpluggedk what do you think?

Reply:
I'm fine with keeping it in PII to be on the safe side.

Reply:
I am with @oldalton here.